news/editions/edition3.md

```diff
@@ -0,0 +1,67 @@
+As contributors and project newcomers alike gear up for the first Spinnaker Gardening Days, we're collecting and discussing [project ideas](https://github.com/spinnaker-hackathon/gardening/wiki/Project-Ideas). Socializing your ideas at this stage will help shape them into projects, so please visit the Wiki and join or create #gardening-idea Slack channels, and discuss with your peers. Alternatively, visit your favorite SIG's channel or meeting, and ask where you can jump in. Don't forget to [register](https://www.eventbrite.com/e/spinnaker-gardening-days-communityhack-tickets-97845696111) to reserve your ticket!
+
+## [How to Build a Centos RPM Via a Spinnaker "Bake"](https://blog.opsmx.com/how-to-build-bake-centos-rpm-using-spinnaker/)
+While Spinnaker doesn't offer a CentOS baking option by default, this tech blog demonstrates the use of package management and S3 to bake a build as a CentOS image, and deploying an instance of the baked image on AWS cloud. With the RPM built, configuring and executing the bake in Spinnaker requires just a few steps.
+
+## ['Just Me and Opensource' Tutorials Available on Youtube](https://www.youtube.com/watch?v=9EUyMjR6jSc&t=2s)
+Fans of the __[Just me and Opensource](https://www.youtube.com/user/wenkatn)__ online tutorials got a nice dose of CD when the popular Youtuber published a new segment of his Kubernetes series focused on deploying Spinnaker to Kubernetes. Brilliant to learn hands-on how these open source software tools and technologies are better together.
+
+## [Netflix's Interactive Slack Notifications for ChatOps with Spinnaker](https://blog.spinnaker.io/interacting-with-spinnaker-via-slack-at-netflix-9ab262e8218d)
+2-way interactive Slack notifications allow Netflix engineers deploying with Spinnaker's new Managed Delivery feature to approve a Spinnaker pipeline manual judgement directly from Slack. Learn how Echo fields communications between notifying services and Slack via interactive notification callbacks.
+
+## [Operating Spinnaker at Scale & the Ops SIG's Mission](https://www.armory.io/blog/scaling-spinnaker-at-salesforce-the-life-of-a-cloud-ops-architect/)
+Discover the mission of the new Operations SIG in this interview with the co-chair, Edgar Magana of Salesforce. This SIG will address several goals and challenges, including reference architecture. Edgar explains these through the lens of Salesforce's Spinnaker implementation, which targets both Kubernetes and EC2 deployments.  
+
+## [Adopting the V2 K8S Provider As V1 Support Sunsets](https://blog.spinnaker.io/farewell-to-the-kubernetes-v1-provider-79d93861c6e4)
+Spinnaker 1.21, expected near the end of June, will be the final release that includes the Kubernetes V1 provider, as community best practices optimize for V2. Read more about generating V2-compliant, version-controlled Kubernetes manifests, and additional steps to migrate pipelines.
```

Comment:
```suggestion
Spinnaker 1.21 will be the final release that includes the Kubernetes V1 provider. Read more about generating V2-compliant, version-controlled Kubernetes manifests, and additional steps to migrate pipelines.
```

Reply:
That 2nd edit there, deleting ", as community best...V2"...
Mainly it's the "best practices optimize for V2" part that doesn't sound right to me. And I figured it's unnecessary. But if you want to keep it that's ok, but maybe instead say "...as best practices increasingly recommend V2" or something like that.